tree.go

```diff
@@ -73,7 +74,7 @@ func (t *Tree) set(key []byte, value []byte) (orphaned []*Node, updated bool) {
 		cmn.PanicSanity(cmn.Fmt("Attempt to store nil value at key '%s'", key))
 	}
 	if t.root == nil {
-		t.root = NewNode(key, value)
+		t.root = NewNode(key, value, 1)
```

Comment:
first node is always version 1?
not t.version?

Remember the case where the empty tree is saved 5 times before the first right, we want version to update each time so it remains in sync with the block height.

Reply:
So if the root is nil, it means it's a brand new tree, and as it starts with 0, it will always be 1, even with t.version+1.

Note that we can't currently save an empty tree, so the first save will always be 1, and then it should increment from there since the root isn't nil anymore. Does that make sense?

Reply:
It makes sense, but I know we found a bug in some test code with this case a while ago.

If you save but t.root == nil, then we still do t.version++ ?
Then make my change here and if we save empty tree 5 times, on save 6 it has a value, it starts at version 6 (to match block 6)?

Maybe odd, but trying to keep in sync with the block height.